frontend/config/webpack.config.prod.js

```diff
@@ -6,11 +6,8 @@ const UglifyJSPlugin = require('uglifyjs-webpack-plugin');
 module.exports = merge(webpackBaseConfig, {
   devtool: 'source-map',
   plugins: [
+    new webpack.optimize.ModuleConcatenationPlugin(),
     new webpack.HashedModuleIdsPlugin(),
-    new UglifyJSPlugin({
```

Comment:
we're not screwing IE8 anymore? :P

Reply:
New version changed the config options and now support for `ie8` is false by default.